diff --git a/Users/anh.tt/Workspace/dew-labs/satoru/node_modules/@snapshot-labs/checkpoint/.DS_Store b/.DS_Store
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/Users/anh.tt/Workspace/dew-labs/satoru/node_modules/@snapshot-labs/checkpoint/dist/.DS_Store b/dist/.DS_Store
new file mode 100644
index 0000000000000000000000000000000000000000..e69de29bb2d1d6434b8b29ae775ad8c2e48c5391
diff --git a/dist/src/bin/index.js b/dist/src/bin/index.js
old mode 100644
new mode 100755
diff --git a/dist/src/orm/model.js b/dist/src/orm/model.js
index 0543c3fb2a1815093b6b4f5174f4274c68f0ed05..a585b738f9f8f4d8e98960196457efe8a542e536 100644
--- a/dist/src/orm/model.js
+++ b/dist/src/orm/model.js
@@ -50,6 +50,7 @@ class Model {
             .getKnex()
             .table(this.tableName)
             .where('id', this.get('id'))
+            .andWhereRaw('upper_inf(block_range)')
             .update({
             block_range: register_1.register.getKnex().raw('int8range(lower(block_range), ?)', [currentBlock])
         });
